cmip6_downscaling/methods/bcsd.py

```diff
@@ -431,9 +449,33 @@ def fit_and_predict(
     return bias_corrected_ds
 
 
+def rechunked_interpolated_prediciton_task_full_time(
+    interpolated_prediction_ds: xr.Dataset, **kwargs
+) -> xr.Dataset:
+    # note: rechunk_zarr_with_caching writes a output store, then xpersist also writes a store.
+    rechunked_interpolated_prediction_ds = rechunk_zarr_array_with_caching(
+        interpolated_prediction_ds, chunking_approach='full_time', max_mem='2GB'
+    )
+    print('rechunked_interpolated_prediction:')
+    print(rechunked_interpolated_prediction_ds)
+    print(rechunked_interpolated_prediction_ds.chunks)
+    return rechunked_interpolated_prediction_ds
+
+
+def rechunked_spatial_anomalies_full_time(spatial_anomalies_ds: xr.Dataset, **kwargs) -> xr.Dataset:
+
+    rechunked_spatial_anomalies = rechunk_zarr_array_with_caching(
+        spatial_anomalies_ds, chunking_approach='full_time', max_mem='2GB'
+    )
+    print('rechunked_spatial_anomalies:')
+    print(rechunked_spatial_anomalies)
+    print(rechunked_spatial_anomalies.chunks)
+    return rechunked_spatial_anomalies
```

Comment:
both of these functions only include one function call in them. Are they needed or could they be replaced inline calls to `rechunk_zarr_array_with_caching`